README.md

```diff
@@ -245,3 +245,82 @@ Suggested [here](https://github.com/whatwg/html/issues/1766#issuecomment-9603285
 Given that `WeakRef`s inevitably expose
 the disentanglement of `MessagePort`s already,
 it does not seem useful to try to hide that.
+
+## TAG Security & Privacy Questionnaire Answers
+
+* **Author:** murakinonoka@google.com
+* **Questionnaire Source:** https://www.w3.org/TR/security-privacy-questionnaire/#questions
+
+### Questions
+
+* **What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?**
+  * This feature exposes that cross-navigation’s destruction
+   or garbage collection has occurred.
+   By passing a  MessagePort to a document and listening to a close event,
+   it is possible to find out
+   when the document has been destroyed
+   or the port has been garbage collected.
+   However, this information is already exposed
+   by polling WeakRef frequently
+   and the `close` event enables this information
+   to be exposed more timely and reliably.
+   In addition to that,
+   the listener cannot know if the close event is caused
+   by document destruction/explicit close/just general GC due to being unreferenced.
+* **Do features  in your specification expose the minimum amount of information necessary to enable their intended uses?**
+  * Yes. Close event needs to be fired
```

Comment:
I would say

Yes. The close event carries no other information except that the port is now closed.